Team6220_2021/src/main/java/org/firstinspires/ftc/team6220_2021/RedParkBlockPLaceLeague2.java

```diff
@@ -4,9 +4,8 @@
 import com.qualcomm.robotcore.hardware.DcMotor;
 import com.qualcomm.robotcore.hardware.DcMotorSimple;
 import com.qualcomm.robotcore.hardware.Servo;
-
 @Autonomous(name = "RedBlockPlace", group = "Autonomous")
-public class RedParkBlockPLaceLeague1 extends MasterOpMode{
+public class RedParkBlockPLaceLeague2 extends MasterOpMode{
     DcMotor motorBackLeft;
```

Comment:
You should not define your motors here.  They should already be defined in MasterOpMode.  Don't define them more than once.  Don't reverse motors here.  Reverse them once, and stick to the same convention throughout all programs.